code/src/components/app-accordion/app-accorion-container/AppAccordionContainer.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from "@testing-library/react";
+import AppAccordionContainer from "@/components/app-accordion/app-accorion-container/AppAccordionContainer";
+
+describe("AppAccordionContainer", () => {
+  test("renders AppAccordionContainer expanded", () => {
+    render(
+      <AppAccordionContainer expanded>
+        <div>Child</div>
+      </AppAccordionContainer>
+    );
+    const accordionElement = screen
+      .getByText("Child")
+      .closest(".spa-accordion-container");
```

Comment:
Is it necessary here ? Can the test be passed without `closest` ?

Reply:
replaced by `parentElement` property